src/OrbitGl/Track.cpp

```diff
@@ -94,6 +94,10 @@ void Track::UpdatePositionOfCollapseToggle() {
   collapse_toggle_->SetWidth(size);
   collapse_toggle_->SetHeight(size);
   collapse_toggle_->SetPos(toggle_pos[0], toggle_pos[1]);
+
+  // This makes sure that changes to the track "collapsible" property are correctly
+  // disabling the triangle toggle, even if they change during runtime.
+  collapse_toggle_->SetIsCollapsible(IsCollapsible());
```

Comment:
Ideally this should be part of Track::SetIsCollapsible() or/and the constructor of collapse toggle. Can you add a TODO?

Reply:
Also, maybe is better renaming the method to Track::UpdateCollapseToggle(), what do you think?

Reply:
Currently, there is not `SetIsCollapsible`, but classes inheriting from Track override `IsCollapsible`. Since the behavior of this may change during runtime, we can't just safely set it once, we have to update it every frame.

And I'd not rename the method, it'll be removed in the next PR anyways.